src/multi-selector.js

```diff
@@ -11,18 +12,20 @@ function DefaultItemComponent(props) {
 	return (
 		<div title={`${getItemTitle(item)}`}>
 			<div
-				className={`${styles['cpr-multi-selector-item__icon']} ${selected ? `cps-bg-primary-green ${styles[`cpr-multi-selector-item__icon--selected`]}` : ""}`}>
-				<i className="cps-icon cps-icon-lg-check" style={{opacity: selected ? "1" : "0"}}></i>
+				className={`${styles['cpr-multi-selector-item__icon']} ${selected
+					? `cps-bg-primary-green ${styles[`cpr-multi-selector-item__icon--selected`]}`
+					: ''}`}>
+				<i className="cps-icon cps-icon-lg-check" style={{ opacity: selected ? '1' : '0' }} />
 			</div>
 			<div className={`${styles['cpr-multi-selector-item__title']}`}>{`${getItemTitle(item)}`}</div>
 		</div>
-	)
+	);
 }
 
 DefaultItemComponent.propTypes = {
-	item: React.PropTypes.any.isRequired,
-	getItemTitle: React.PropTypes.func,
-	selectedItems: React.PropTypes.array.isRequired
+	item: PropTypes.any.isRequired,
```

Comment:
these are literally the only substantive changes within this file. All others are prettier doing its thing.